### PR TITLE
Add 'say' command in console

### PR DESCRIPTION
**Pull request by telk5093**
Add 'say' command to send chat message in console.
(+ spacing behind "}")

Please notice me if any problem, and note that I'm new to git and c/c++.

### CODE REVIEW

**Comment by janisozaur** [comment]
It may not get looked into until you fix the CIs.

**Comment by Gymnasiast** [comment]
More specifically, it's the formatting CI that fails. See its output here: https://travis-ci.org/OpenRCT2/OpenRCT2/jobs/455268028

File: src/openrct2/interface/InteractiveConsole.cpp [hunk 1]
```diff
@@ -1306,7 +1307,30 @@ static int32_t cc_save_park(
...
+static int32_t cc_say(
+    [[maybe_unused]] InteractiveConsole& console, [[maybe_unused]] const utf8** argv, [[maybe_unused]] int32_t argc)
+{
+    if (network_get_mode() == NETWORK_MODE_NONE || network_get_status() != NETWORK_STATUS_CONNECTED
+        || network_get_authstatus() != NETWORK_AUTH_OK)
+    {
+        console.WriteFormatLine("Not work in client mode.");
+        return 0;
+    }
+    else {
```
Comment (Lastorder-DC) [inline src/openrct2/interface/InteractiveConsole.cpp]:
```suggestion
    else
    {
```

File: src/openrct2/interface/InteractiveConsole.cpp [hunk 2]
```diff
@@ -1306,7 +1307,30 @@ static int32_t cc_save_park(
...
+        }
+        else
+        {
+            console.WriteFormatLine("Input your message");
+            return 0;
+        }
+    }
+}
+
+using console_command_func = int32_t(*)(InteractiveConsole& console, const utf8** argv, int32_t argc);
```
Comment (Lastorder-DC) [inline src/openrct2/interface/InteractiveConsole.cpp]:
```suggestion
using console_command_func = int32_t(*) (InteractiveConsole& console, const utf8** argv, int32_t argc);
```

**Comment by telk5093** [comment]
Thanks for your comments and force-pushing, I didnt' noticed my code's style was wrong.

File: src/openrct2/interface/InteractiveConsole.cpp [hunk 3]
```diff
@@ -1308,6 +1309,29 @@ static int32_t cc_save_park(
     return 1;
 }
 
+static int32_t cc_say(InteractiveConsole& console, const utf8** argv, int32_t argc)
+{
+    if (network_get_mode() == NETWORK_MODE_NONE || network_get_status() != NETWORK_STATUS_CONNECTED
+        || network_get_authstatus() != NETWORK_AUTH_OK)
+    {
+        console.WriteFormatLine("Not work in client mode.");
```
[the remark this answers is not in the document]
Comment (telk5093) [reply src/openrct2/interface/InteractiveConsole.cpp]: If you mean "client mode" is not connected to server, may I ask you if any reason for use "say" in client mode?

[the remark this answers is not in the document]
Comment (Gymnasiast) [reply src/openrct2/interface/InteractiveConsole.cpp]: What I mean is that the error message on this line is not correct English.

Do you mean that the command only works if you're connected to a server? And can anyone that is connected to a server issue this command?

[the remark this answers is not in the document]
Comment (telk5093) [reply src/openrct2/interface/InteractiveConsole.cpp]: 1. Fixed. Thanks for noticing me.
 1. Yes, I have thought there is no reason using "say" command when I'm not connected to a server.
 1. Yes, for everyone. By their user name. Is there any something wrong?

[the remark this answers is not in the document]
Comment (Gymnasiast) [reply src/openrct2/interface/InteractiveConsole.cpp]: In that case, could you change the error message to
`This command only works in multiplayer mode.` ?

'Client mode' implies being connected to a server.

[the remark this answers is not in the document]
Comment (telk5093) [reply src/openrct2/interface/InteractiveConsole.cpp]: Ok. Sorry for conflict and lack of my english. I'll change it.

**Comment by parthi1234** [comment]
Ok

File: src/openrct2/interface/InteractiveConsole.cpp [hunk 4]
```diff
@@ -1386,29 +1410,30 @@ static constexpr const utf8* console_window_table[] = {
...
-    { "quit", cc_close, "Closes the console.", "quit"},
-    { "remove_park_fences", cc_remove_park_fences, "Removes all park fences from the surface", "remove_park_fences"},
+    { "quit", cc_close, "Closes the console.", "quit" },
+    { "remove_park_fences", cc_remove_park_fences, "Removes all park fences from the surface", "remove_park_fences" },
     { "remove_unused_objects", cc_remove_unused_objects, "Removes all the unused objects from the object selection.", "remove_unused_objects" },
     { "reset_user_strings", cc_reset_user_strings, "Resets all user-defined strings, to fix incorrectly occurring 'Chosen name in use already' errors.", "reset_user_strings" },
     { "rides", cc_rides, "Ride management.", "rides <subcommand>" },
-    { "save_park", cc_save_park, "Save current state of park. If no name specified default path will be used.", "save_park [name]"},
+    { "save_park", cc_save_park, "Save current state of park. If no name specified default path will be used.", "save_park [name]" },
+    { "say", cc_say, "Say to other players.", "say [message]" },
```
Comment (janisozaur) [inline src/openrct2/interface/InteractiveConsole.cpp]: `[]` signifies optional argument, but the message is not. Use `<>` instead.

Comment (telk5093) [reply src/openrct2/interface/InteractiveConsole.cpp]: Thanks. Done.

**Comment by Broxzier** [comment]
This should be added to the changelog.

**Pull request merged**